lld/test/ELF/linkerscript/section-class.test

```diff
@@ -310,17 +310,19 @@ SECTIONS {
 .byte 0x12, 0x34
 
 .section .b,"aM",@progbits,1
+.p2align 1
```

Comment:
An alignment of 2 works, but a larger alignment makes the effect clearer.

Reply:
We also need to dump the section content (e.g. `-x second` )

Reply:
Done.